playwright/integrations.test.ts

```diff
@@ -67,35 +70,7 @@ describe("webhooks", () => {
 
     // if we change the shape of our webhooks, we can simply update this by clicking `u`
     // console.log("BODY", body);
-    expect(body).toMatchInlineSnapshot(`
-    Object {
-      "createdAt": "[redacted/dynamic]",
-      "payload": Object {
-        "additionInformation": "[redacted/dynamic]",
-        "attendees": Array [
-          Object {
-            "email": "test@example.com",
-            "name": "Test Testson",
-            "timeZone": "[redacted/dynamic]",
-          },
-        ],
-        "description": "",
-        "destinationCalendar": null,
-        "endTime": "[redacted/dynamic]",
-        "metadata": Object {},
-        "organizer": Object {
-          "email": "pro@example.com",
-          "name": "Pro Example",
-          "timeZone": "[redacted/dynamic]",
-        },
-        "startTime": "[redacted/dynamic]",
-        "title": "30min between Pro Example and Test Testson",
-        "type": "30min",
-        "uid": "[redacted/dynamic]",
-      },
-      "triggerEvent": "BOOKING_CREATED",
-    }
-  `);
+    expect(JSON.stringify(body)).toMatchSnapshot(`webhookResponse.txt`);
```

Comment:
Extracted to snapshot file

Reply:
Cool!

Also noticed we can do [Visual comparisons](https://playwright.dev/docs/test-snapshots)!

Reply:
@KATT that's on the TODOs for sure.